### DISCUSSION TITLE
Hyperion and Android TV on one Raspberry Pi

### DISCUSSION THREAD

**Discussion by wlad031** [hyperion-project/hyperion.ng#1681] [General]
Hi dear community,

Will it be possible to run Hyperion on LineageOS based on Android? My idea is to run both Android TV and Hyperion on one Raspberry Pi, however, I haven't found information on how to install it.

**Comment by Lord-Grey** [comment]
There is nothing out of the box for installation currently.
You might check in the Hyperion forum, if someone successfully build Hyperion on Android.
